scrapers/sis_scraper/sis_scraper_test.py

```diff
@@ -100,6 +100,7 @@ def bitpack_section_conflict(bp1, bp2):
 # courses. This compares the bitpack results to the results from neieve comparision
 def test_conflict_bitpacks():
     for term in os.listdir("data"):
+        print(f"Testing {term}")
```

Comment:
I think you forgot to remove this

Reply:
I personally think its useful to see what term it failed on. Before this I didn't know it was winter specific.

Reply:
My bad. I got confused when I saw "testing" and though you left some testing code in the main problem. I didn't realize it was the testing script